config/configureEnv.js

```diff
@@ -39,7 +40,8 @@ const configureEmail = async () => {
       return email;
     }
     const answer = await askFor("email");
-    return answer;
+    process.env.NODE_CONFIG_APP_EMAIL = answer.email;
+    return answer.email;
```

Comment:
`const { email } = await askFor('email')` ? definitely just a stylistic preference, nbd

Reply:
yep was meaning to do this, just forgot before push